pkg/validate/validate.go

```diff
@@ -147,6 +148,14 @@ func ClusterName(v string) error {
 	return validateSubdomain(v)
 }
 
+// ClusterNameIBMRestrictions checks if the given string is a valid name for a cluster on IBM Cloud and returns an error if not
+func ClusterNameIBMRestrictions(v string) error {
+	if !ibmCloudClusterNameRegex.MatchString(v) {
```

Comment:
Do we need a regex for this? Or can we just validate the name length? I think the character restrictions are already handled by `validate.ClusterName`.

Reply:
Sure, I will simplify the check to be length only, and leave the character restrictions to the `validate.ClusterName` instead.